ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86.yaml

```diff
@@ -93,8 +93,8 @@ tests:
     - chain: create-infra-move-ingress-monitoring-registry
     workflow: openshift-qe-node-density-heavy
   timeout: 5h0m0s
-- as: perfscale-aws-data-path-9nodes-periodic
-  cron: 0 21 * * 2
+- always_run: false
+  as: perfscale-aws-data-path-9nodes-periodic
```

Comment:
self-managed aws cluster should still work as normal, as they have the port enabled already.

Reply:
@mukrishn We had 4 runs of aws since yesterday and 3 out of 4 failed with timeout issues. Here are the runs:
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/44169/rehearse-44169-pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.13-nightly-x86-perfscale-aws-data-path-9nodes-periodic/1711775642418155520 
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/44169/rehearse-44169-pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.13-nightly-arm-perfscale-aws-data-path-9nodes-arm-periodic/1711794694440095744
https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-nightly-x86-perfscale-aws-data-path-9nodes-periodic/1711849152037523456
https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.13-nightly-x86-perfscale-aws-data-path-9nodes-periodic/1711849150305275904

Reply:
Looks like even the self-managed ones are failed, I guess they need the SG rule as well. 
CC: @venkataanil

Reply:
@mukrishn In that case, do you want to proceed and merge this including turning off self managed aws?